src/components/previewers/VideoPlayer.jsx

```
@@ -0,0 +1,180 @@
+                    )}
+                  </div>
+                </div>
+              </div>
+              <div className="w-full h-[1px] bg-gray-200 my-2"></div>
+              <div className="action-buttons-container pl-3">
+                <div className="flex items-center gap-4">
+                  <button
+                    onClick={handleLike}
+                    className="action-button"
+                    aria-label="Like"
+                  >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)